docs/Getting Started.md

```diff
@@ -0,0 +1,778 @@
+6. Show usage of CLI helpers like `status`, `list` etc.
+7. Show how to deal with http ingress and push some data to the application
+8. Verify that the application works
+9. Undeploy the application
+
+
+## Download and install the CLI
+
+cloudflow CLI can be downloaded from [lightbend public bintray repository](https://bintray.com/lightbend/cloudflow-cli). Download the version appropriate for your platform and install it in your local system. Please make sure you have the executable in your path with proper permission settings.
+
+## Develop a sample cloudflow application
+
+In this section we will develop a sample application that will feature the main components of cloudflow. It will be a simple hello-world style application, expressive enough to demonstrate the major features of cloudflow. It will not have many of the advanced features that cloudflow offers - the main idea is to give you a feel for what it takes to build a complete application and deploy it in running condition in a GKE cluster.
+
+We will develop a simple pipeline that processes events from a wind turbine farm. The application will receive streaming data that will pass through the following transformations :
+
+* ingested by a streamlet (ingress)
+* processed by a streamlet to convert into a domain object (processor)
+* validated by a streamlet (splitter) that separates valid and invalid records
+* logged in to loggers to be checked at output (egress)
+
+Each of the above terminologies (ingress, processor, splitter and egress) are explained in the [Concepts Guide](https://blank.com). Here's an overview of the application pipeline architecture:
+
+![Application Pipeline](images/pipe.001.jpeg?raw=true "Application Pipeline")
+
+One of the important features of cloudflow architecture is the complete separation of the components from how they are connected as part of the pipeline. The _streamlets_ described above are the individual building blocks of the pipeline. You can connect them using a declarative language that forms the _blueprint_ of the pipeline. Streamlets can be shared across blueprints making them individual reusable objects. And just like streamlets, blueprints also form an independent component of a cloudflow application.
+
+### Project structure
+
+Here's how we would structure a typical cloudflow application as a project.
+
+```
+   |-project
+   |-src
+   |---main
+   |-----avro
+   |-----blueprint
+   |-----resources
+   |-----scala
+   |-------sensordata
+   |-build.sbt
+```
+
+This is a Scala project and we have the following structural components at the leaf level of the above tree:
+
+* **avro** : contains the avro schema of the domain objects
+* **blueprint** : contains the blueprint of the application in a file named `blueprint.conf`
+* **scala** : contains the source code of the application under the package name `sensordata`
+* **build.sbt** : the sbt build script
+
+### The sbt build script
+
+Here's a minimal version of the sbt build script:
+
+```
```

Comment:
I think you would want to add **build.sbt :** at the top as you did with the other files for consistency reasons.

Reply:
done ..